src/compositions/useWeb3.ts

```diff
@@ -16,7 +16,7 @@ const getWeb3 = (): Web3 => {
 };
 
 const getChainId = async (): Promise<number> => {
-  return await web3.eth.getChainId();
+  return await web3?.eth?.getChainId();
```

Comment:
Is this optional chaining useful? We either load `web3` or not I believe, there is no scenario when we have `web3` but without `eth` namespace.

Reply:
I added that when I got an error that eth was undefined when I was trying to get chainId for useLspFactory, but I'll remove it, since I'm no longer using it.